docsets/openFrameworks/docset.json

```diff
@@ -9,9 +9,13 @@
     "aliases": ["of"],
 
     "specific_versions": [
+		{
+			"version": "0.10.0",
+			"archive": "versions/0.10.0/openFrameworks.tgz"
```

Comment:
You need to add an archive at this path as well.

Reply:
ok. I copied to that path as well.